src/about-menu/about-menu.html

```diff
@@ -83,7 +83,7 @@
         <p class="links">
           <a href="https://github.com/Automattic/studio" target="_blank">GitHub</a>
           <span class="separator">&middot;</span>
-          <a href="https://github.com/Automattic/studio/issues/new/choose" target="_blank">Report a bug</a>
+          <a href="https://github.com/Automattic/studio/issues/new/choose" target="_blank">Share Feedback</a>
```

Comment:
It would be great if we translate the strings of the About screen. However, this would imply refactoring to render React components, as currently, it's a plain HTML view.

Reply:
As a follow-up, I created [a GitHub issue](https://github.com/Automattic/dotcom-forge/issues/7522) to address this.